translations/messages+intl-icu.de.yml

```diff
@@ -2096,7 +2096,9 @@ last.used: "Zuletzt genutzt:"
 layer.land-useplan.public: "Planzeichnung"
 layer.land-useplan.short: "B-Plan"
 layer.land-useplan: "Bebauungsplan"
+layer.no.longer.included.in.map.service: "Der zuvor gespeicherte Layer ist im aktuellen Kartendienst nicht mehr enthalten. Möglicherweise wurde er entfernt oder umbenannt."
```

Comment:
Maybe we could shorten the key to `layer.unavailable_in_service`, or just `layer.unavailable`? The latter is less specific, but for now it doesn't seem to collide with any other use case